src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -993,6 +999,9 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
             success = _dispatch->SetCursorColor(color);
             TermTelemetry::Instance().Log(TermTelemetry::Codes::OSCRCC);
             break;
+        case OscActionCodes::Hyperlink:
+            success = _dispatch->AddHyperlink(uri);

```

Comment:
This isn't the right place, necessarily, for this comment... but at any point does anything actually validate that the URL is really a URL? Or does it just store the arbitrary string and let `ShellExecute` all the way at the end choke if it's invalid?

If it's that... then maybe we should be checking the return code from `ShellExecute` and reporting what/why it didn't work?

Reply:
We check for http or https schemes now!